bin/all_sky_search/pycbc_template_kde_calc

```diff
@@ -52,44 +59,50 @@ def optimizedparam(sampleval, nfold=2):
     maxFOM = FOM[(optbw, optalpha)]
     return optbw, optalpha
 
+assert len(args.fit_param) == len(args.log_param)
```

Comment:
Can you move this line to just after `init_logging` to make it clearer that it is input checking